### ISSUE TITLE
4.1 release breaks oauth with "Could not verify the security token on this form"

### ISSUE THREAD

**Issue by recipedude** [matomo-org/plugin-GoogleAnalyticsImporter#162]
Was working until 4.1 plugin auto-updated.

When trying to "Re-authorize", behaviour as expected, sends to Google for re-oauth flow, after selecting "Allow" the redirect back to Matomo pukes with:

```
An error occurred
Could not verify the security token on this form.
```

Redirect url is:

https://mato.self.hosted/index.php?module=GoogleAnalyticsImporter&action=processAuthCode&state=c7redacted74&code=4/0Aredacted4A&scope=https://www.googleapis.com/auth/analytics.readonly

Moving through the UI there seems to be no obvious way to revert back to a previous version. 

Also now broken:

Google Search Console - configured accounts
 Google Search Console - configured websites

With:
An error occured while validating the account connection: { "error": "invalid_grant", "error_description": "Bad Request" }

**Comment by diosmosis** [comment]
Hi @recipedude, thanks for the report and sorry the error is affecting you, we're aware of this problem and are looking for a fix currently.

**Comment by recipedude** [comment]
Cool, get that PR merged and roll out release 4.1.2... :)

**Comment by diosmosis** [comment]
Hi @recipedude, thanks for your comment and sorry you are waiting, we are working on the next release and will release it when it is ready.
